### PR TITLE
a profileEdit RPC that properly busts CardCache

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

**Comment by maxtaco** [comment]
r? @patrickxb 

cc: @cjb

File: go/engine/profile_edit.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+func (e *ProfileEdit) Run(ctx *Context) (err error) {
+	defer e.G().CTrace(ctx.NetContext, "ProfileEdit#Run", func() error { return err })()
+	_, err = e.G().API.Post(libkb.APIArg{
+		Endpoint:    "profile-edit",
+		NeedSession: true,
+		Args: libkb.HTTPArgs{
+			"bio":      libkb.S{Val: e.arg.Bio},
+			"fullname": libkb.S{Val: e.arg.FullName},
```
Comment (cjb) [inline go/engine/profile_edit.go]: Is there a bug here?  The profile-edit API endpoint takes `full_name`, not `fullname`.  And if so, how come your tests pass?

**Comment by maxtaco** [comment]
Whoops!  Thanks for the review.  Fixed, with better testing....  It was previously only testing `location`.

**Comment by maxtaco** [comment]
Oh, and it looks like I also forgot to `git add` my test.

**Comment by maxtaco** [comment]
Cool, let's just wait for C/I and then G2G.  sorry about that bug.

**Pull request merged**